tests/test_emailer.py

```diff
@@ -311,13 +312,14 @@ def test_send(self):
 
     def test__send_email(self):
         """Verify that send_email calls certain methods on smtplib.SMTP."""
-        integration = self._integration()
+        _ = self._integration()
         emailer = Emailer.from_sitewide_integration(self._db)
-        mock = MockSMTP()
-        emailer._send_email("you@library", "email body", mock)
+        mock_class = MockSMTP
```

Comment:
Would it be too much work to use a `MagicMock` instead?

Reply:
Probably not. I'll take a look.